sample/SampleAspNetCoreApp/wwwroot/lib/bootstrap/dist/js/npm.js

```diff
@@ -1,13 +1,13 @@
 // This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
-require('../../js/transition.js')
-require('../../js/alert.js')
-require('../../js/button.js')
-require('../../js/carousel.js')
-require('../../js/collapse.js')
-require('../../js/dropdown.js')
-require('../../js/modal.js')
-require('../../js/tooltip.js')
-require('../../js/popover.js')
-require('../../js/scrollspy.js')
-require('../../js/tab.js')
-require('../../js/affix.js')
\ No newline at end of file
+require('../../js/transition.js');
```

Comment:
Code cleanup also touches `js` files. I reckon we should not check this in and leave these changes as is. cc @elastic/dotnet

Reply:
+1